README.md

```diff
@@ -904,6 +906,34 @@ We then run both the seed and the package by executing the following command:
 dbt build --select package:dbt_project_evaluator dbt_project_evaluator_exceptions
 ```
 
+----
+## Running this package as a CI check
+
+Once you have addressed all current misalignments in your project (either by fixing them or configuring exceptions), you can use this package as a CI check to ensure code changes don't introduce new misalignments. The setup will vary based on whether you are using dbt CLoud or dbt core, but the general steps are as follows:
+
+### 1. Override test severity with an environment variable
+
+By default the tests in this package are configured with "warn" severity, we can override that for our CI jobs with an environment variable:
+1. Create an environment variable to define the appropriate severity for each environment. In dbt Cloud, for example, we can easily create an environment variable `DBT_PROJECT_EVALUATOR_SEVERITY` that is set to "error" for the Continuous Integration environment and "warn" for all other environments:
```

Comment:
Could we mention that it is also possible with env vars for dbt Core, but the actual implementation will depend on how dbt is run/orchestrated?